introduction.typ

```diff
@@ -8,4 +8,4 @@ I'm Carson McManus and I'm a 5/5 undergrad Software Engineer. I've written a lot
 
 === Chris Roddy
 
-I'm Chris, a 4/4 Software Engineering major and minoring in computer science. I might add more to this later.
\ No newline at end of file
+I'm Chris, a 4/4 Software Engineering major and minoring in computer science. I'm interested in all areas of development and outside of programming, I really enjoy cooking.
```

Comment:
Technically not a related change but I'll let it slide this time. Try to avoid making unrelated changes because it bloats your PR and makes it harder to review and merge.